examples/hello_debugger.cpp

```diff
@@ -61,9 +90,15 @@ class Debugger {
 
   Debugger(const EventHandler&);
 
-  // run() instructs the debugger to continue execution.
+  // runs the debugger's main loop
```

Comment:
nit: The documentation pattern I've been using for functions / methods is `verb() does blah...`
Also super-dooper-uber-nit, fullstops.

Reply:
No problem, can dp.